codecov.threshold

```diff
@@ -24,6 +24,7 @@ istio.io=5
 istio.io/istio/mixer/adapter/solarwinds/internal/papertrail/papertrail_logger.go=50
 istio.io/istio/pilot/pkg/config/memory/monitor.go=30
 istio.io/istio/pilot/pkg/proxy/envoy/v2=10
+istio.io/istio/pilot/pkg/proxy/envoy/v2/rds.go=25
```

Comment:
25 seems low.  The failures I have seen have been at 50%.

You are correct that it is non-deterministic.  The file is so short that if there are no errors when the server closes it is flagged as a code coverage error.